doc/test.md

```diff
@@ -102,7 +102,17 @@ For riscv64: `qemu-riscv64 -L /usr/riscv64-linux-gnu <TestBinary>`
 
    Install lcov `sudo apt-get install lcov`.
 
-   Build cases.
+   Build cases with `-DGCOV=ON`.
+
+   ```
+   cd libspdm
+   mkdir build
+   cd build
+   cmake -DARCH=<x64|ia32|arm|aarch64|riscv32|riscv64|arc> -DTOOLCHAIN=GCC -DTARGET=<Debug|Release> -DCRYPTO=<mbedtls|openssl> -DGCOV=ON..

```

Comment:
Need space after -DGCOV=ON
`-DGCOV=ON ..`

Reply:
I have updated it.  I appreciate your advise!  Thanks!